combined.js

```diff
@@ -0,0 +1,159 @@
+// ICEBOX - gives user the option to play either game
+
+const { reverse } = require("dns");
+const readline = require("readline");
+const { start } = require("repl");
+const rl = readline.createInterface(process.stdin, process.stdout);
+
+function ask(questionText) {
+  return new Promise((resolve, reject) => {
+    rl.question(questionText, resolve);
+  });
+}
+
+combined()
+
+async function combined() {
+    console.log("Hello! We have two game options: Either A) You think of a number and the computer tries to guess it, or B) The computer thinks of a number and you try to guess it. Which would you like to play?")
+    let game = await ask("Please input A or B => ")
```

Comment:
Currently, "B" is not utilized and your program will accept any input to mean the player wants the reverse game.